src/index.jsx

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter as Router } from 'react-router-dom'
 
 import App from './app';
 import './styles/app.scss';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Router>
```

Comment:
why this change? https://flaviocopes.com/react-router-uselocation-usehistory-undefined/